copy/copy.go

```diff
@@ -162,6 +168,10 @@ func Image(policyContext *signature.PolicyContext, destRef, srcRef types.ImageRe
 		return err
 	}
 
+	if options == nil {
+		options = &Options{}
+	}
```

Comment:
Fair enough, that works :) If we are doing it this way, can you move it to the top eliminate the three now redundant `options != nil` checks as well, please?